analyzer-application/src/main/resources/application.properties.example

```diff
@@ -0,0 +1,4 @@
+#Compile time configuration for the application server
+
+#Uncomment if you want to configure the port the application runs on. The default is 8080
+#server.port
```

Comment:
Can we change this to "#server.port=[###]" so it's extra clear how to specify the port number?